dateparser/__init__.py

```diff
@@ -39,6 +40,11 @@ def parse(date_string, date_formats=None, languages=None, locales=None, region=N
         Configure customized behavior using settings defined in :mod:`dateparser.conf.Settings`.
     :type settings: dict
 
+    :param detect_languages_function:
+        A function for language detection that takes as input a `text` (the `date_string`) and
```

Comment:
`` `text` `` does not refer to a variable name, right? perhaps we mean
```suggestion
        A function for language detection that takes as input a string (the `date_string`) and
```